dataflowengine/src/main/scala/io/shiftleft/dataflowengine/language/TrackingPoint.scala

```diff
@@ -94,12 +94,12 @@ class TrackingPoint(raw: GremlinScala[nodes.TrackingPoint]) extends NodeSteps[no
   private def getTrackingPoint(vertex: Vertex): Option[nodes.TrackingPoint] = {
     vertex match {
       case identifier: nodes.Identifier =>
-        getTrackingPoint(identifier._astIn.nextChecked)
+        getTrackingPoint(identifier._argumentIn().nextChecked)
```

Comment:
I recently noticed these new _foo methods. Where are they coming from? When did we introduce them? I'm not a fan of this, we should rather define these kinds of methods as part of the query language.

Reply:
Bernhard implemented those a few weeks back. They have been introduced as replacement for the low level gremlin API `vertices(<direction>, <edgeTypes>)` in order to increase performance.